<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>744</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>594</width>
    <height>744</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cadastra Cliente</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(230, 230, 230);</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="txtCelular">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="inputMask">
     <string>##-#####-####</string>
    </property>
    <property name="maxLength">
     <number>13</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBairro">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>450</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCpf">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="inputMask">
     <string>#############</string>
    </property>
    <property name="maxLength">
     <number>13</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>551</width>
      <height>671</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Cliente </string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nome</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>RG</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CPF</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>210</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Celular</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>270</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Telefone </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>320</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Logradouro</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtLogradouro">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>320</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="maxLength">
      <number>40</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtRg">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="inputMask">
      <string>############</string>
     </property>
     <property name="maxLength">
      <number>12</number>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBuscarCliente">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>530</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 216, 98);
border-style: outset;
border-width: 2px;
border-radius: 15px; 
border-color: rgb(126, 126, 126);
</string>
     </property>
     <property name="text">
      <string>Consultar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPetcliente">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>530</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 216, 98);
border-style: outset;
border-width: 2px;
border-radius: 15px; 
border-color: rgb(126, 126, 126);</string>
     </property>
     <property name="text">
      <string>Adicionar Pet</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCidade">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>470</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>360</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cep</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>410</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Bairro</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>470</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cidade</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCep">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>370</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="inputMask">
      <string>#####-###</string>
     </property>
     <property name="maxLength">
      <number>9</number>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCadastrar">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>530</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 216, 98);
border-style: outset;
border-width: 2px;
border-radius: 15px; 
border-color: rgb(126, 126, 126);</string>
     </property>
     <property name="text">
      <string>Cadastrar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNome">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtFixo">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>270</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="inputMask">
      <string>##-####-####</string>
     </property>
     <property name="maxLength">
      <number>12</number>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label</zorder>
    <zorder>label_12</zorder>
    <zorder>label_7</zorder>
    <zorder>label_6</zorder>
    <zorder>txtLogradouro</zorder>
    <zorder>txtRg</zorder>
    <zorder>btnBuscarCliente</zorder>
    <zorder>btnPetcliente</zorder>
    <zorder>txtCidade</zorder>
    <zorder>label_5</zorder>
    <zorder>label_8</zorder>
    <zorder>label_10</zorder>
    <zorder>txtCep</zorder>
    <zorder>btnCadastrar</zorder>
    <zorder>txtNome</zorder>
    <zorder>txtFixo</zorder>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>txtCelular</zorder>
   <zorder>txtBairro</zorder>
   <zorder>txtCpf</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtNome</tabstop>
  <tabstop>txtRg</tabstop>
  <tabstop>txtCpf</tabstop>
  <tabstop>txtCelular</tabstop>
  <tabstop>txtFixo</tabstop>
  <tabstop>txtLogradouro</tabstop>
  <tabstop>txtCep</tabstop>
  <tabstop>txtBairro</tabstop>
  <tabstop>txtCidade</tabstop>
  <tabstop>btnCadastrar</tabstop>
  <tabstop>btnBuscarCliente</tabstop>
  <tabstop>btnPetcliente</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
